ui/src/pages/dashboard/index.module.scss

```diff
@@ -1,93 +1,30 @@
 @import 'src/scss/lib';
 
-.container {
-  display: flex;
-  height: 100vh;
-  position: relative;
-  width: 100vw;
+.header-container {
+  border-bottom: 1px solid $color-button-border;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  height: 3em;
+  margin-bottom: 7em;
+  padding: 1em;
+  width: 100%;
 }
 
-.sidebar {
-  align-items: center;
-  border-right: 1px solid $color-button-bg;
-  display: flex;
-  flex-direction: column;
-  position: relative;
-  width: 220px;
-}
-
-.header {
-  align-items: center;
-  display: flex;
-  margin: 9px 21px 0 20px;
-  width: 179px;
-}
-
-.dot-menu-icon {
-  color: $color-text-dim2;
-  cursor: pointer;
-  height: 22px;
-  margin-left: 12px;
-  width: 22px;
-}
-
-.detail {
-  align-items: center;
-}
-
-.logo-image {
-  border-radius: 50%;
-  height: 20px;
-  width: 20px;
-}
-
-.pic-container {
-  display: flex;
-  flex-direction: row-reverse;
-  height: 18px;
-  margin-left: auto;
-  width: 18px;
-
-  .pic {
-    border-radius: 50%;
-  }
-
-  .online-status {
-    align-self: flex-end;
-    background: $color-online;
-    border: 1px solid $color-bg-dark;
-    border-radius: 50%;
-    height: 8px;
-    margin-left: 1px;
-    margin-top: 1px;
-    position: absolute;
-    width: 8px;
-  }
+.header-title {
+  color: $color-text-primary;
+  font-size: 0.8em;
+  line-height: 1em;
+  text-align: left;
 }
 
-.footer {
+.plus-icon {
+  border: 1px solid $color-cta;
+  border-radius: 4px;
+  box-shadow: 1px 2px 0 rgba(0, 0, 0, 0.4);
+  box-sizing: border-box;
+  color: $color-cta;
   cursor: pointer;
-  display: flex;
-  margin-bottom: 24px;
-  margin-top: auto;
-}
-
-.feedback-icon {
-  background: url('/assets/images/feedback-icon.svg') 0 0 no-repeat;
-  height: 12px;
-  margin-right: 8px;
-  width: 12px;
-}
-
-.feedback-text {
-  background-color: transparent;
-  border: 0;
-  color: $color-text-primary;
-  font-size: 12px;
-  justify-self: flex-end;
-  line-height: 10px;
-
-  &:hover {
-    color: $color-text-bright1;
-  }
+  height: 1.5em;
+  margin-right: 2em;
+  width: 1.5em;
```

Comment:
These look like they are leftover from styling Header. I suggest we remove them, and extract Header to its own component, and more appropriately name it  (e.g `TopBar`).